blockchain/blockchain-wallet-v4-frontend
# Feat(Ability to unarchive BSV wallets)

## Description
- Adds ability to unarchive BSV wallets
- Synchronizes the UX for all archived wallets in all coin tables (see screenshot below)
- Add/update unit tests

## Screenshot
<img width="629" alt="screen shot 2019-01-02 at 10 44 26 am" src="https://user-images.githubusercontent.com/6364918/50602466-f2938b00-0e7c-11e9-95b1-b1144be0bc4a.png">

## Change Type
- Feature
- Bug Fix

## Testing Steps
- Login with account that has archived BTC wallet, see BSV wallet is also unarchived. Unarchive it.

## Code Checklist
